### PR TITLE
adds inVerification field

### PR DESCRIPTION
**Pull request by katelynsills**
## Description
 Adds `inVerification` field to the pregnancy center schemas, updates the field to be the userId when /verify is called. Includes a script to clear the `inVerification` field (set to `null` as needed). The `inVerification` field is cleared on user logout, and when the pregnancy Center record is updated. 

### Ensuring that `inVerification` gets set to the userId when a pregnancy center is pulled for verification

1. Click the "Get One Resource" button.
2. Look at the `inVerification` field for the resource in the database

### Ensuring that the script sets the `inVerification` field to null on all pregnancy centers

1. Follow the above steps to make sure `inVerification` has been set
2. Run `yarn clear-iv`
3. Search database for `{  inVerification: {   $ne: null  } }` (this should be empty, as all are null)

### Ensuring that updating the pregnancyCenter clears the `inVerification` field
1. Follow the above steps to make sure `inVerification` has been set
2. hit submit to save/update a pregnancy center
3.  Look at the `inVerification` field for the resource in the database. It should gone back to null

### Ensuring that logging out clears the `inVerification` fields for the user and only the user
1. Follow the above steps to make sure `inVerification` has been set, maybe do this multiple times
2. Go into the database and change one of the pregnancy center's `inVerification` field to be something that is not the UserId
3. Logout
4. Check that the only `inVerification` left that isn't null is the one you changed, and the ones that previously had the UserId are now null

### CODE REVIEW

**Comment by katelynsills** [comment]
Ok, I think this is ready for another review. Also, tests pass on Travis CI now.

**Comment by katelynsills** [comment]
This was all ready to go, but it looks like one test is failing. I can't reproduce it locally, so I'm guessing it has something to do with the node version, since that's the only change between my local environment and the Travis CI environment. Maybe once we merge in the node version upgrade, I'll merge master in again and see. Right now, though, I'm at a loss.

**Comment by katelynsills** [comment]
Yay! We're passing now!

File: server/pregnancy-centers/schema/mongoose-schema.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const pregnancyCenterSchema = mongoose.Schema({
 	address: addressSchema,
 	email: String,
 	hours: Object,
+	inVerification: String,
```
Comment (karenmrose) [inline server/pregnancy-centers/schema/mongoose-schema.js]: This should be `ObjectId`

**Comment by katelynsills** [comment]
Added in Joi validation for MongoDB ObjectIds

**Pull request merged**